lib/phoenix_component.ex

```diff
@@ -1401,14 +1404,22 @@ defmodule Phoenix.Component do
         def about_page(assigns)
       end
 
-  Multiple invocations of `embed_templates` is also supported.
+  Multiple invocations of `embed_templates` is also supported, which can be
+  useful if you have more than one template format. For example:
+
+      defmodule MyAppWeb.Emails do
+        use Phoenix.Component
+
+        embed_templates "emails/*.html", suffix: "_html"
+        embed_templates "emails/*.text", suffix: "_text"
+      end
   """
   defmacro embed_templates(pattern, opts \\ []) do
     quote do
       Phoenix.Template.compile_all(
-        &Phoenix.Component.__embed__/1,
+        &Phoenix.Component.__embed__(&1, unquote(opts)[:suffix] || ""),
         Path.expand(unquote(opts)[:root] || __DIR__, __DIR__),
-        unquote(pattern) <> ".html"
+        unquote(pattern)
```

Comment:
This should not be changed. :)

```suggestion
        unquote(pattern) <> ".html"
```

Reply:
Oh, you want to explicitly embed `.text` ones. So maybe we also need a `:format` option? /cc @chrismccord